models/articles.model.js

```diff
@@ -13,3 +13,32 @@ exports.fetchArticleById = async (article_id) => {
   }
   return { article: article.rows[0] };
 };
+
+exports.fetchArticles = async (article_id) => {
```

Comment:
We shouldn't be querying the database twice here, lets have a go at achieving this in one query using joins and aggregate functions 👍